### DISCUSSION TITLE
How to run nextjs project? I keep getting error saying Global CSS cannot be imported

### DISCUSSION THREAD

**Discussion by glaksmono** [vercel/next.js#28902] [Help]
I'm getting the following error:

    ./styles/globals.scss
    Global CSS cannot be imported from files other than your Custom <App>. 
    Due to the Global nature of stylesheets, and to avoid conflicts, 
    Please move all first-party global CSS imports to pages/_app.js. 
    Or convert the import to Component-Level CSS (CSS Modules).

    Read more: https://nextjs.org/docs/messages/css-global
    Location: pages/_app.tsx

[![enter image description here][1]][1]

And this error disappears when I commented out the following lines in the `_app.tsx`

```
// import "@styles/globals.scss";
```

[![enter image description here][2]][2]

But noticed that the stylesheet isn't being applied. Any ideas how to fix this?

Here's the output of my `npm list`

```
 $ npm list --depth 0
├── @types/react@17.0.20
├── eslint-config-next@11.1.2
├── eslint@7.32.0
├── next@11.1.2
├── react-dom@17.0.2
├── react@17.0.2
├── sass@1.39.0
└── typescript@4.4.2
```

```
 $ npm list -g --depth 0
/Users/glaksmono/.nvm/versions/node/v16.8.0/lib
└── npm@7.22.0
```

  [1]: https://i.stack.imgur.com/aXW5I.png
  [2]: https://i.stack.imgur.com/AwXnY.png

**Comment by glaksmono** [comment]
I also tried fresh following this : https://github.com/vercel/next.js/tree/canary/packages/create-next-app

but I'm still getting the same error 

```
 $ npx create-next-app hello-world
Creating a new Next.js app in /glaksmono/Documents/Sandbox/hello-world.

Using npm.

Installing dependencies:
- react
- react-dom
- next

npm WARN deprecated querystring@0.2.1: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.

added 319 packages, and audited 320 packages in 13s

48 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Installing devDependencies:
- eslint
- eslint-config-next


added 225 packages, and audited 545 packages in 15s

81 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Initialized a git repository.

Success! Created hello-world at /glaksmono/Documents/Sandbox/hello-world
Inside that directory, you can run several commands:

  npm run dev
    Starts the development server.

  npm run build
    Builds the app for production.

  npm start
    Runs the built app in production mode.

We suggest that you begin by typing:

  cd hello-world
  npm run dev


 ~/glaksmono/Documents/Sandbox                   ✔  757  18:01:50
 $ cd hello-world

 ~/glaksmono/Documents/Sandbox/hello-world   ma ✔  758  18:02:01
 $ npm run dev

> hello-world@0.1.0 dev
> next dev

ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
(node:7198) [DEP_WEBPACK_MODULE_ISSUER] DeprecationWarning: Module.issuer: Use new ModuleGraph API
(Use `node --trace-deprecation ...` to show where the warning was created)
error - ./styles/globals.css
Global CSS cannot be imported from files other than your Custom <App>. Due to the Global nature of stylesheets, and to avoid conflicts, Please move all first-party global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://nextjs.org/docs/messages/css-global
Location: pages/_app.js
```

**Reply by glaksmono** [reply]
I tried to follow this tutorial : https://nextjs.org/learn/basics/assets-metadata-css/global-styles

and I'm getting the same error, as soon as I added the following lines in the `_app.js`

```
import '../styles/global.css'
```

**Reply by dawkaka** [reply]
Is '../styles/global.css' the path to the global css in your working directory or './styles/global.css' ? If it's the later, move your styles directory from the pages directory to the top-level directory.

**Reply by glaksmono** [reply]
Hi @Yousiph1 - I attached how I structure the project. Anything I'm missing here?

![Screen Shot 2021-09-08 at 19 17 03](https://user-images.githubusercontent.com/286254/132507830-5eeb365a-66b7-4fb7-92fa-9c67f1e05156.png)

**Comment by leerob** [comment]
Related RFC: https://github.com/vercel/next.js/discussions/27953

**Comment by ilmanurh** [comment]
Hi, I got the same error after installing Next JS. Have you already found any solution? Thanks in advance.

**Comment by mertefeisikgor** [comment]
Hey mate, what happens if you don't use sass,scss, and just import it as .css, sass is a problematic at times so that could be first to cleared out
